tictactoe.Tests/GameTest.cs

```diff
@@ -85,5 +83,117 @@ public void GetCurrentPlayerReturnsPlayer1Test()
 
             Assert.Equal("X", currentPlayer._marker);
         }
+
+        [Fact]
+        public void GamePlayChecksForValidMoveTest1()
+        {
+            board.cells = new string[] {
+                "X", "O", " ",
+                " ", " ", " ",
+                " ", " ", " "};
+
+            console.setUserInputs(new List<string> { "1", "1", "2", "2", "3", "4", "5", "6", "7", "8", "9" });
+
+            game = new Game(console, rules);
+
+            game.SetBoard(board);
+            game.SetPlayers(player1, player2);
+
+            game.Menu();
+
+            Assert.Equal("O", board.cells[1]);
+        }
+
+        [Fact]
+        public void GamePlayChecksForValidMoveTest2()
+        {
+            board.cells = new string[] {
+                "X", "O", "X",
+                " ", " ", " ",
+                " ", " ", " "};
+
+            console.setUserInputs(new List<string> { "1", "1", "2", "3", "4", "5", "6", "7", "8", "9" });
+
+            game = new Game(console, rules);
+
+            game.SetBoard(board);
+            game.SetPlayers(player1, player2);
+
+            game.Menu();
+
+            Assert.Equal("O", board.cells[3]);
+        }
+
+        [Fact]
+        public void GamePlayChecksForDraw()
+        {
+            board.cells = new string[] {
+                "X", "O", "X",
+                "X", "O", "O",
+                "O", "X", "X"};
+
+            console.setUserInputs(new List<string> { "1", "1", "2", "3", "4", "5", "6", "7", "8", "9" });
+
+            game.SetBoard(board);
+            game.SetPlayers(player1, player2);
+
+            game.Menu();
+
+            Assert.True(rules.DrawIsCalled == true);
+        }
+
+        [Fact]
+        public void GamePlayChecksForWon()
+        {
+            board.cells = new string[] {
+                "X", "O", "X",
+                "O", "X", "O",
+                "X", "O", "X"};
+
+            console.setUserInputs(new List<string> { "1", "1", "2", "3", "4", "5", "6", "7", "8", "9" });
+
+            game.SetBoard(board);
+            game.SetPlayers(player1, player2);
+
+            game.Menu();
+
+            Assert.True(rules.WonIsCalled == true);
```

Comment:
`Assert.True` already asserts whether the statement is true or not, why do we need `== true`?

Reply:
commit 32ef40b1deed3f1c2644b459df331be1ed5feaa8